camunda-commons-ui/lib/analytics/modal.html

```diff
@@ -34,13 +34,19 @@ <h3>{{'TELEMETRY_DETAILS_HEADER' | translate}}</h3>
             <li>
                 <b>{{'TELEMETRY_DETAILS_META_DATA' | translate}}</b>
                 <p>{{'TELEMETRY_DETAILS_META_DATA_WHAT' | translate}}</p>
+                <i>{{'TELEMETRY_DETAILS_META_DATA_WHY' | translate}}</i>
             </li>
             <li>
                 <b>{{'TELEMETRY_DETAILS_USAGE_STATISTICS' | translate}}</b>
                 <p>{{'TELEMETRY_DETAILS_USAGE_STATISTICS_WHAT' | translate}}</p>
+                <i>{{'TELEMETRY_DETAILS_USAGE_STATISTICS_WHY' | translate}}</i>
             </li>
         </ul>
-        <i>{{'TELEMETRY_DETAILS_META_DATA_WHY' | translate}}</i>
+
+        {{'TELEMETRY_DETAILS_DOCUMENTATION_1' | translate}} <a
+            href="https://docs.camunda.org/manual/latest/introduction/telemetry/"
```

Comment:
Can we link to the respective version?

Reply:
I was thinking about that, however we don't expose the current version to the Webapp globally. We could manually replace it for maintenance branches manually, but this introduces maintenance overhead for future versions.

I'm a bit lost as to where to go from here, as exposing the package version as an env variable in browserify is a bigger refactoring which I don't consider part if this ticket. We also don't have staged info for the [provacy notice](https://camunda.com/legal/privacy/), so maybe we can create a follow-up ticket on this. 

What do you think?

Reply:
I agree with your assessment. Please create a follow-up ticket.